requests/models.py

```diff
@@ -813,13 +831,17 @@ def iter_lines(self, chunk_size=ITER_CHUNK_SIZE, decode_unicode=None, delimiter=
             #
             # If we're using `splitlines()`, we only do this if the chunk
             # ended midway through a line.
-            incomplete_line = lines[-1] and lines[-1][-1] == chunk[-1]
+            incomplete_line = lines[-1] and lines[-1][-1] and lines[-1][-1] == chunk[-1]
```

Comment:
Sorry I forgot to remove them. It should be prevented by the check for chunk[1:].